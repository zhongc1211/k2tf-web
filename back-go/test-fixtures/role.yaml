apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  name: elasticsearch
rules:
  - apiGroups:
      - ""
    resources:
      - endpoints
    verbs:
      - get
