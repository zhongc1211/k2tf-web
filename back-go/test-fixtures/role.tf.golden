resource "kubernetes_role" "elasticsearch" {
  metadata {
    name = "elasticsearch"
  }
  rule {
    verbs      = ["get"]
    api_groups = [""]
    resources  = ["endpoints"]
  }
}
